contrib/grpc_util/src/test/java/io/opencensus/contrib/grpc/util/StatusConverterTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.grpc.util;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link StatusConverter}. */
+@RunWith(JUnit4.class)
+public class StatusConverterTest {
+
+  @Test
+  public void convertFromGrpcStatus() {
+    // Without description
+    for (io.grpc.Status.Code grpcCode : io.grpc.Status.Code.values()) {
+      io.grpc.Status grpcStatus = io.grpc.Status.fromCode(grpcCode);
+      io.opencensus.trace.Status opencensusStatus = StatusConverter.fromGrpcStatus(grpcStatus);
+      assertThat(opencensusStatus.getCanonicalCode().toString())
+          .isEqualTo(grpcStatus.getCode().toString());
```

Comment:
Suggestion: I think this line could use `grpcCode` instead of `grpcStatus.getCode()`.  The same comment applies to the three other assertions on codes.

Reply:
I feel like the tests are for Status -> Status so I think writing expectation using the status makes more sense. Let me know if you feel strong about this.

Reply:
I don't feel strongly about it; I just meant it as a suggestion.